src/ConnectedMode/Binding/SolutionBindingOperation.cs

```diff
@@ -57,12 +57,6 @@ public void SaveRuleConfiguration(IEnumerable<IBindingConfig> bindingConfigs, Ca
                     return;
                 }
 
-                foreach (var solutionItem in config.SolutionLevelFilePaths)
-                {
-                    var ruleSetDirectoryPath = Path.GetDirectoryName(solutionItem);
-                    fileSystem.Directory.CreateDirectory(ruleSetDirectoryPath); // will no-op if exists
```

Comment:
Previously, the only real value left in the `SolutionBindingOperation` class was that it ensured a directory exists before writing a rule file. This PR moves that responsibility to the individual binding config implementations, so this class can then be deleted.